python/ray/serve/tests/test_metrics.py

```diff
@@ -31,21 +33,19 @@ def verify_metrics(do_assert=False):
 
         expected_metrics = [
             # counter
-            "num_router_requests_total",
-            "num_http_requests_total",
-            "deployment_queued_queries_total",
-            "deployment_request_counter_requests_total",
-            "deployment_worker_starts_restarts_total",
+            "serve_num_router_requests",
+            "serve_num_http_requests",
+            "serve_deployment_queued_queries",
+            "serve_deployment_request_counter",
+            "serve_deployment_replica_starts",
             # histogram
             "deployment_processing_latency_ms_bucket",
             "deployment_processing_latency_ms_count",
             "deployment_processing_latency_ms_sum",
             # gauge
             "replica_processing_queries",
```

Comment:
This metric is now called `"serve_replica_processing_queries"`. Can we change this string to that?